File: C/Lancement Programme/3.1.c
#include <stdio.h>

int main(int argc , char ** argv)
{ 
    if (argc == 3)
    {
        // Convertir d'abord les chaines de caracteres en entiers 
        int a = atoi(argv[1]);
        int b = atoi(argv[2]); 
        
        int s = a + b ;
        printf(" %d + %d = %d\n", a , b , s);

        return (0); 
    }

    else 
    {
        printf("Usage : %s nombre1 nombre2 " , argv[0]);
        return (1);
    }

}

File: Python/Lancement de Programme/3.1.py
import sys 

if len(sys.argv) == 3:
    a = int(sys.argv[1])
    b = int(sys.argv[2])

    s = a + b 
    print(f"{a} + {b} = {s}")

else :
    print(f"Usage : {sys.argv[0]} nombre1 nombre2")